webpack.config.js

```diff
@@ -49,6 +49,28 @@ module.exports = {
       },
       {
         test: /\.css$/,
+        exclude: [
+          path.resolve(__dirname, 'src/css/styles.css'),
+          path.resolve(__dirname, 'src/css/diff.css')
+        ],
+        use: [
+          { loader: 'style-loader' },
+          {
+            loader: 'css-loader',
```

Comment:
I think we need to make sure `postcss-loader` comes over here, too, otherwise we will mysteriously lose autoprefixer support as we migrate styles to modules. (Plus I’m sure we’ll eventually want to add other PostCSS plugins as we get more serious about CSS.)

Reply:
Good catch! I will add it to the new pipeline.